content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -152,6 +161,36 @@ kubectl delete -k manifests/kustomize/env/dev
 # kubectl delete -k manifests/kustomize/env/gcp
 ```
 
+## Best practices maintaining custom manifests
+
+### Maintain a repo for your manifests
+
+Save the following to a source controlled repo.
+
+File `kustomization.yaml`.
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+# Edit the following to change the deployment to your custom namespace.
+namespace: kubeflow
+# You can add other customizations here using kustomize.
+# Edit ref in the following link to deploy a different version of Kubeflow Pipelines.
+bases:
+- github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref={{% kfp-latest-version %}}
+```
+
+### How to
```

Comment:
Should this entire section disappear now that we have an upgrade section above? If we need to keep this section, then we should explain how it differs from the earlier upgrade section - is it for upgrades when using a custom manifest?

Reply:
Looks good!